src/main/kotlin/no/nav/helsearbeidsgiver/domene/inntektsmelding/v1/skjema/SkjemaInntektsmelding.kt

```diff
@@ -113,11 +115,37 @@ private fun validerRefusjonMotInntekt(
                 vilkaar = refusjon.endringer.all { it.beloep <= inntekt.beloep },
                 feilmelding = Feilmelding.REFUSJON_OVER_INNTEKT,
             ),
+            // "Fallback"-sjekk dersom ingen AGP - da skal dato for refusjonEndring alltid være senere enn InntektDato
+            valider(
+                vilkaar = refusjon.endringer.all { it.startdato.isAfter(inntekt.inntektsdato) },
+                feilmelding = Feilmelding.REFUSJON_ENDRING_FOER_INNTEKTDATO,
+            ),
+
         )
     } else {
         emptyList()
     }
 
+/*
+Endring i refusjon skal alltid ha dato etter AGP (dersom det er AGP).
+ */
+private fun validerRefusjonMotAgp(refusjon: Refusjon?, agp: Arbeidsgiverperiode?): List<FeiletValidering> {
+    val agpMax = agp?.let {
+            arbeidsgiverperiode ->
+        arbeidsgiverperiode.perioder.maxOfOrNull { it.tom }
+    }
```

Comment:
Kunne denne vært
```suggestion
    val agpMax = agp?.perioder?.maxOfOrNull { it.tom }
```
?